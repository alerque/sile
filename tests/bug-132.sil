\begin[papersize=a6]{document}
\nofolios
\script[src=packages/verbatim]
\define[command=verbatim:font]{\font[family=Hack]}
\font[family=Hack]
\begin{verbatim}
#          #
##();:,i[]##
##MMWWQQGG##
\end{verbatim}
\font[family=Gentium Plus]
\begin{verbatim}
#          #
##();:,i[]##
##MMWWQQGG##
\end{verbatim}
\end{document}
