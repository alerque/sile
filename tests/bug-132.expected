Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	28.1541
Set font 	Hack;10;400;;normal;;LTR
T	591	(#)
Mx 	81.1075
T	591	(#)
Mx 	14.8819
My 	38.1424
T	591	(#)
Mx 	20.9024
T	591	(#)
Mx 	26.9229
T	629	(()
Mx 	32.9434
T	630	())
Mx 	38.9639
T	598	(;)
Mx 	44.9844
T	583	(:)
Mx 	51.0049
T	584	(,)
Mx 	57.0254
T	158	(i)
Mx 	63.0460
T	623	([)
Mx 	69.0665
T	624	(])
Mx 	75.0870
T	591	(#)
Mx 	81.1075
T	591	(#)
Mx 	14.8819
My 	49.8592
T	591	(#)
Mx 	20.9024
T	591	(#)
Mx 	26.9229
T	56 56 103 103 77 77 32 32	(MMWWQQGG)
Mx 	75.0870
T	591	(#)
Mx 	81.1075
T	591	(#)
Mx 	14.8819
My 	66.7459
T	591	(#)
Mx 	81.1075
T	591	(#)
Mx 	14.8819
My 	76.7342
T	591	(#)
Mx 	20.9024
T	591	(#)
Mx 	26.9229
T	629	(()
Mx 	32.9434
T	630	())
Mx 	38.9639
T	598	(;)
Mx 	44.9844
T	583	(:)
Mx 	51.0049
T	584	(,)
Mx 	57.0254
T	158	(i)
Mx 	63.0460
T	623	([)
Mx 	69.0665
T	624	(])
Mx 	75.0870
T	591	(#)
Mx 	81.1075
T	591	(#)
Mx 	14.8819
My 	88.4510
T	591	(#)
Mx 	20.9024
T	591	(#)
Mx 	26.9229
T	56 56 103 103 77 77 32 32	(MMWWQQGG)
Mx 	75.0870
T	591	(#)
Mx 	81.1075
T	591	(#)
End page
Finish
